CHANGELOG/CHANGELOG-1.3.md

```diff
@@ -1,3 +1,7 @@
+# v1.3.5 - Changelog since v1.3.4
```

Comment:
we should also capture this in the master branch changelog https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/master/CHANGELOG/CHANGELOG-1.3.md